server/src/test/scala/services/export/StreamingDesksExportSpec.scala

```diff
@@ -0,0 +1,269 @@
+package services.`export`
+
+import actors.minutes.MockMinutesLookup
+import actors.queues.DateRange
+import akka.NotUsed
+import akka.stream.scaladsl.{Sink, Source}
+import drt.shared.CrunchApi.{CrunchMinute, MinutesContainer, StaffMinute}
+import drt.shared.Terminals.T1
+import drt.shared.{Queues, TM, TQM}
+import services.SDate
+import services.crunch.CrunchTestLike
+import services.exports.StreamingDesksExport
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class StreamingDesksExportSpec extends CrunchTestLike {
+
+  "Given CrunchMinutes and StaffMinutes on 1 day" >> {
+
+    val pax = 5
+
+    val eeaDeskRec = 5
+    val nonEEADeskRec = 4
+    val eGateRec = 3
+
+
+    val eeaDeskDep = 4
+    val nonEEADeskDep = 3
+    val eGateDep = 2
+
+    val waitTime = 8
+    val depDesk = 9
+    val depWait = 10
+    val actDesk = 11
+    val actWait = 12
+    val minute1 = SDate("2020-11-19T00:00")
+    val minute2 = SDate("2020-11-19T00:15")
+
+    val crunchMinutesContainer = MinutesContainer[CrunchMinute, TQM](List(
+      CrunchMinute(T1, Queues.EeaDesk, minute1.millisSinceEpoch, pax, 6, eeaDeskRec, waitTime, Option(eeaDeskDep), Option(depWait), Option(actDesk), Option(actWait)),
```

Comment:
Do you want to also extract the `6` here to a val since all the other values are extracted?